llvmlite/tests/test_binding.py

```diff
@@ -1264,7 +1274,7 @@ def test_lookup_current_process_symbol_fails(self):
         # enabled searching the current process for symbols.
         msg = 'Failed to materialize symbols:.*getversion'
         with self.assertRaisesRegex(RuntimeError, msg):
-            self.jit(asm_getversion, "getversion")
+            self.jit(asm_getversion, "getversion", suppress_errors=True)
```

Comment:
Q: Why doesn't all raised error print to stderr? I thought we'll need `supress_errors=True` everywhere.

Reply:
LLVM's error handling is now designed like this: there are two important classes `Error` and `Expected<T>`.  `Expected<T>` is basically Rust's `Result<T, Error>`. Some LLVM functions return these values and any that I get I turn into Python exceptions. Internally, LLVM doesn't always propagate them though. For instance, the object layer initializer has to return `std::unique_ptr<ObjectLayer>`, so you have to eat the errors. If you let an error be destroyed without "checking" it (printing it somewhere), then it will trigger a panic. There's `cantFail` that does the equivalent of Rust's `Result::unwrap`. It's also possible to push an error into an `ExecutionSession` and let it propagate through an error handler there. That's what I'm doing with `suppress_errors`; it's hiding errors that have been pushed into the `ExecutionSession` rather than printing them to console. Only the errors I can't explicitly handle end up there though. This only needs to be set at the time the execution session is created.